src/library/autodj/autodjprocessor.cpp

```diff
@@ -280,7 +280,7 @@ void AutoDJProcessor::fadeNow() {
 
         if (toDeckCurrentSecond >= introStart &&
                 toDeckCurrentSecond <= introEnd &&
-                introEnd >= 0) {
+                introEnd >= 0 && introStart != introEnd) {
```

Comment:
remove the `introEnd >= 0` as it does not work as intended. Also add a comment explaining why introStart would equal introEnd

Reply:
explaining that in the commit message is good, but in this case it should be in a code comment so anyone reading the code sees it

Reply:
I have updated the code and added the comment.